django_prometheus/middleware.py

```diff
@@ -222,7 +222,10 @@ def process_request(self, request):
         if request.META.get("HTTP_X_REQUESTED_WITH") == "XMLHttpRequest":
             self.label_metric(self.metrics.requests_ajax, request).inc()
 
-        content_length = int(request.META.get("CONTENT_LENGTH") or 0)
+        content_length = 0
+        raw_content_length = request.META.get("CONTENT_LENGTH", "")
+        if raw_content_length.isdigit():
```

Comment:
I wouldn't bother with the `isdigit()` call, but use `try..except ValueError, TypeError` instead: `isdigit` would return `True` for digits that `int()` would still fail to convert, for example `₂`. It is unlikely that this would happen in real world, but it is theoretically possible.

I would also add a unit test for this change.